  D3D12Application.cpp
C:\Users\k024g\OneDrive\ドキュメント\GitHub\DX12_test\D3D12Application.cpp(275,39): warning C4267: '=': 'size_t' から 'UINT' に変換しました。データが失われているかもしれません。
  (ソース ファイル '/D3D12Application.cpp' をコンパイルしています)
  
  main.cpp
  Modeldata.cpp
C:\Users\k024g\OneDrive\ドキュメント\GitHub\DX12_test\Modeldata.cpp(75,53): warning C4267: '=': 'size_t' から 'UINT' に変換しました。データが失われているかもしれません。
  (ソース ファイル '/Modeldata.cpp' をコンパイルしています)
  
C:\Users\k024g\OneDrive\ドキュメント\GitHub\DX12_test\Modeldata.cpp(76,49): warning C4267: '=': 'size_t' から 'UINT' に変換しました。データが失われているかもしれません。
  (ソース ファイル '/Modeldata.cpp' をコンパイルしています)
  
C:\Users\k024g\OneDrive\ドキュメント\GitHub\DX12_test\Modeldata.cpp(77,41): warning C4267: '=': 'size_t' から 'UINT' に変換しました。データが失われているかもしれません。
  (ソース ファイル '/Modeldata.cpp' をコンパイルしています)
  
  Texture.cpp
  Win32Application.cpp
  コードを生成中...
Texture.obj : error LNK2019: 未解決の外部シンボル "public: void __cdecl Texture::Term(void)" (?Term@Texture@@QEAAXXZ) が関数 "public: __cdecl Texture::~Texture(void)" (??1Texture@@QEAA@XZ) で参照されました
Texture.obj : error LNK2019: 未解決の外部シンボル "private: struct D3D12_SHADER_RESOURCE_VIEW_DESC __cdecl Texture::GetViewDesc(bool)" (?GetViewDesc@Texture@@AEAA?AUD3D12_SHADER_RESOURCE_VIEW_DESC@@_N@Z) が関数 "public: bool __cdecl Texture::Init(struct ID3D12Device *,class DescriptorPool *,struct D3D12_RESOURCE_DESC const *,bool)" (?Init@Texture@@QEAA_NPEAUID3D12Device@@PEAVDescriptorPool@@PEBUD3D12_RESOURCE_DESC@@_N@Z) で参照されました
C:\Users\k024g\OneDrive\ドキュメント\GitHub\x64\Debug\DX12_test.exe : fatal error LNK1120: 2 件の未解決の外部参照
